### ISSUE TITLE
overwriting Mousetrap.stopCallback?

### ISSUE THREAD

**Issue by jedierikb** [ccampbell/mousetrap#288]
In v1.4, we could overwrite Mousetrap.stopCallback by

    Mousetrap.stopCallback = function(e, element, combo) {
    ...
    }

This approach does not work in v1.5+.  What is the recommended way to provide custom behavior?

**Comment by stillnet** [comment]
I am having the same problem...

**Comment by bunkat** [comment]
Looks like he moved `stopCallback` to the Mousetrap prototype.  Was able to override it by doing this:

````
Mousetrap.prototype.stopCallback = function(e, element, combo) {
    return false;
}
````

**Comment by ccampbell** [comment]
Yeah thanks @bunkat I should have commented.  You can see an example here

https://github.com/ccampbell/mousetrap/blob/master/plugins/global-bind/mousetrap-global-bind.js#L14-L26

**Comment by franciscop** [comment]
This fixed it also for me, but I preferred to create a local instance instead of overwriting the default behaviour:

```js
Editor.prototype.shortcuts = function(){
  
  // Make it local and overwrite default
  var mousetrap = new Mousetrap();
  mousetrap.stopCallback = function(){ return false; };
  
  // ...
};
```
